python_modules/libraries/dagster-hightouch/dagster_hightouch/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1!0+dev"
```

Comment:
I spot checked a couple other libs but might've missed it: do our libs need a `version.py`?

Reply:
oh jk you're right. I just looked at the wrong dir